src/vm/clarity.rs

```diff
@@ -1561,9 +1561,9 @@ mod tests {
     pub fn test_post_condition_failure_contract_publish() {
         use chainstate::stacks::db::*;
         use chainstate::stacks::*;
+        use net::strings::StacksString;
         use util::hash::Hash160;
         use util::secp256k1::MessageSignature;
-        use util::strings::StacksString;
```

Comment:
Not sure to follow, if we are moving `StacksString` to `net`, then `net` becomes a vm dependency.

Reply:
`net` was already a vm dependency as `vm` uses `StacksMessageCodec`and `NetError`. This is a step towards removing `util` as a dependency of `net`. 

I agree that introducing more coupling between `net` and `vm` goes against the ultimate goal of decoupling `vm`... Do you have any other proposals for where these things should live?